.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,7 +18,6 @@ Before we can merge this PR, please make sure that all the following items have
 checked off. If any of the checklist items are not applicable, please leave them but
 write a little note why.
 
-- [ ] Targeted PR against correct branch (`manta` or `manta-pc`) with right title (start with [Manta] or [Manta-PC]),
```

Comment:
@stechu @Dengjianping 

We only have the `manta` branch so this doesn't make sense.
Maybe we can classify by runtime ?

Reply:
we do have `dolphin` branch for testnet.